codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2HeadersDecoder.java

```diff
@@ -33,10 +33,18 @@
 import static io.netty.handler.codec.http2.Http2Exception.connectionError;
 
 public class DefaultHttp2HeadersDecoder implements Http2HeadersDecoder, Http2HeadersDecoder.Configuration {
+    private static final float HEADER_ARRAY_SIZE_ALPHA = 1 / 5;
```

Comment:
Maybe name these something like `HEADERS_COUNT_WEIGHT_NEW` and `HEADERS_COUNT_WEIGHT_HISTORICAL`?